assets/src/edit-story/components/library/panes/media/media3p/stories/media3pCategories.js

```diff
@@ -37,7 +37,7 @@ const Container = styled.div`
 export const _default = () => {
   return (
     <Container>
-      <Media3pCategories />
+      <Media3pCategories categories={[]} />
```

Comment:
Please add some fake categories in here.

Reply:
Done.